pymc3/distributions/bart.py

```diff
@@ -63,22 +63,30 @@ def __init__(
             )
         self.m = m
         self.alpha = alpha
-        self.y_std = Y.std()
-
-        if scale is None:
-            self.leaf_scale = NormalSampler(sigma=None)
-        elif isinstance(scale, (int, float)):
-            self.leaf_scale = NormalSampler(sigma=Y.std() / self.m ** scale)
 
         if inv_link is None:
-            self.inv_link = lambda x: x
+            self.inv_link = self.link = lambda x: x
+            self.init_mean = self.Y.mean()
+            self.Y_un = self.Y
+        elif isinstance(inv_link, str):
+            if inv_link == "logistic":
+                self.inv_link = expit
+                self.link = lambda x: (x - 0.5) * 10
```

Comment:
Shouldn't the link be the inverse of the expit?

Reply:
In principle you are right, but that will return -inf, inf. And also the purpose of self.link is just to allow the sampler to propose reasonable values for the leaf nodes. So this approximation seems to work ok.

Reply:
btw, no need for apologies. Actually my first attempt was to try something less ad-hoc than this.